### ISSUE TITLE
Error creating account

### ISSUE THREAD

**Issue by larrysalibra** [blockstack/blockstack-portal#392]
A user on our slack reports problems creating an account:

```
Uncaught TypeError: _blockstack.PrivateKeychain is not a constructor
    at http://localhost:8888/js/main.js:7425:25
    at http://localhost:8888/js/main.js:107360:18
    at Object.initializeWallet (http://localhost:8888/js/main.js:107439:12)
    at WelcomeModal.createAccount (http://localhost:8888/js/main.js:2429:20)
    at Object.invokeGuardedCallback (http://localhost:8888/js/main.js:86486:5)
    at executeDispatch (http://localhost:8888/js/main.js:80553:21)
    at Object.executeDispatchesInOrder (http://localhost:8888/js/main.js:80573:7)
    at executeDispatchesAndRelease (http://localhost:8888/js/main.js:79978:22)
    at executeDispatchesAndReleaseTopLevel (http://localhost:8888/js/main.js:79989:10)
    at Array.forEach (native)
```

**Comment by larrysalibra** [comment]
This should be fixed in `v0.6.2`. please re-open if it still happens.

**Comment by jeffslofish** [comment]
I just did a clean install today, and I am getting this when I attempt to create an account:

````
account.js:235 Uncaught TypeError: _blockstack.PrivateKeychain is not a constructor
    at http://localhost:3000/js/main.js:7430:25
    at http://localhost:3000/js/main.js:109697:18
    at Object.initializeWallet (http://localhost:3000/js/main.js:109776:12)
    at WelcomeModal.createAccount (http://localhost:3000/js/main.js:2432:20)
    at Object.ReactErrorUtils.invokeGuardedCallback (http://localhost:3000/js/main.js:87710:16)
    at executeDispatch (http://localhost:3000/js/main.js:81701:21)
    at Object.executeDispatchesInOrder (http://localhost:3000/js/main.js:81721:7)
    at executeDispatchesAndRelease (http://localhost:3000/js/main.js:81126:22)
    at executeDispatchesAndReleaseTopLevel (http://localhost:3000/js/main.js:81137:10)
    at Array.forEach (native)````
